src/containers/DayEntries.js

```diff
@@ -42,7 +41,7 @@ const mapDispatchToProps = dispatch => ({
     fetchEntries: (s, e) => dispatch(fetchEntries(s, e))
 })
 
-const mapStateToProps = (state) => {
+const mapStateToProps = (state, props) => {
```

Comment:
In case we want to pass custom properties for the component that are not managed by redux, for example the router properties we can use this 2nd argument that redux passes to us to be able to do that, see: https://redux.js.org/advanced/usagewithreactrouter#reading-from-the-url

Reply:
I didn't do that here to keep things simple now